src/services/base.service.js

```diff
@@ -83,13 +83,14 @@ const qb = new QueryBuilder({
 const baseCreate = async ({ req }, resourceType) => {
   logger.info(`${resourceType} >>> create`);
   checkContentTypeHeader(req.headers);
-  checkProvenanceHeader(req.headers);
-  const res = req.res;
   const data = req.body;
   //Create a new id regardless of whether one is passed
   data['id'] = uuidv4();
-  populateProvenanceTarget(req.headers, res, [{ reference: `${resourceType}/${data.id}` }]);
-
+  if (Object.keys(req.headers).includes('x-provenance')) {
```

Comment:
Nitpick: we can check the key with an `O(1)` lookup rather than having to do a search in an array with `includes` (`O(n)`)

```suggestion
  if (req.headers['x-provenance']) {
```